dashboard/app/controllers/rubrics_controller.rb

```diff
@@ -1,9 +1,9 @@
 class RubricsController < ApplicationController
   include Rails.application.routes.url_helpers
 
-  before_action :require_levelbuilder_mode_or_test_env, except: [:submit_evaluations, :get_ai_evaluations, :get_teacher_evaluations, :ai_evaluation_status_for_user, :ai_evaluation_status_for_all, :run_ai_evaluations_for_user, :run_ai_evaluations_for_all]
-  load_resource only: [:get_teacher_evaluations, :ai_evaluation_status_for_user, :ai_evaluation_status_for_all, :run_ai_evaluations_for_user, :run_ai_evaluations_for_all]
-  load_and_authorize_resource except: [:submit_evaluations, :get_ai_evaluations, :get_teacher_evaluations, :ai_evaluation_status_for_user, :ai_evaluation_status_for_all, :run_ai_evaluations_for_user, :run_ai_evaluations_for_all]
+  before_action :require_levelbuilder_mode_or_test_env, except: [:submit_evaluations, :get_ai_evaluations, :get_teacher_evaluations, :get_teacher_feedback, :ai_evaluation_status_for_user, :ai_evaluation_status_for_all, :run_ai_evaluations_for_user, :run_ai_evaluations_for_all]
+  load_resource only: [:get_teacher_evaluations, :get_teacher_feedback, :ai_evaluation_status_for_user, :ai_evaluation_status_for_all, :run_ai_evaluations_for_user, :run_ai_evaluations_for_all]
+  load_and_authorize_resource except: [:submit_evaluations, :get_ai_evaluations, :get_teacher_evaluations, :get_teacher_feedback, :ai_evaluation_status_for_user, :ai_evaluation_status_for_all, :run_ai_evaluations_for_user, :run_ai_evaluations_for_all]
```

Comment:
it seems important to properly authorize the new route since it could return PII. Please add auth tests like these and then make sure they pass: https://github.com/code-dot-org/code-dot-org/blob/1f7fb6c9aade6892401913edd1425d4cad6b108e/dashboard/test/controllers/rubrics_controller_test.rb#L39-L41

ideally the auth rules would go in ability.rb, but if you're not familiar with those then it seems fine to continue the existing pattern in this file with rules like these: https://github.com/code-dot-org/code-dot-org/blob/71f304d563ec0d13481e3785430eb503e85bcf8d/dashboard/app/controllers/rubrics_controller.rb#L141

Reply:
Is that done for `get_teacher_evaluations`?

Reply:
Sounds good! I just did what the evaluations path did and assumed it was good! what you are saying makes 100% sense... though we might just look at doing an initial POST to the feedback api path instead of combining it in the one submit call.